.cookiecutter/includes/.github/workflows/environments.json

```diff
@@ -0,0 +1,30 @@
+{
+  "QA": {
+    "github_environment_url": "https://qa-via.hypothes.is/https://en.wikipedia.org/api/rest_v1/page/pdf/Comet_Kohoutek",
+    "aws_region": "us-west-1",
+    "elasticbeanstalk_application": "via",
+    "elasticbeanstalk_environment": "qa"
+  },
+  "QA_Edu": {
```

Comment:
Casing looks a bit of an odd duck here. As this is only an ID and the name take precedence in the UI would this be better as `qa_edu`, or `QAEdu`?

Now might be a good time to revisit the base file, as I see we have `CI` in there as opposed to `ci` or `continuous-integration`?